website/blog/releases/0.8.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Apache Fluss 0.8: Streaming Lakehouse with Iceberg/Lance"
```

Comment:
Add `Announcing` in title will be more precise for an announcement, how about: `Announcing Apache Fluss 0.8: Streaming Lakehouse for AI & Analytics` ?